src/components/PageHeading.tsx

```diff
@@ -27,7 +31,15 @@ function PageHeading({
       <div className="max-w-4xl ml-0 2xl:mx-auto">
         {breadcrumbs ? <Breadcrumbs breadcrumbs={breadcrumbs} /> : null}
         <H1 className="mt-0 text-primary dark:text-primary-dark -mx-.5 break-words">
-          {title}
+          <div className="flex items-center">
```

Comment:
probably don't want this to be flex (if it's a long title, the icon should probably be on the last line)